ui/components/MesheryMeshInterface/PatternServiceForm.js

```diff
@@ -89,8 +101,12 @@ function PatternServiceForm({ formData, schemaSet, onSubmit, onDelete, reference
         return (
           <div className={classes.formWrapper}>
             <AppBar className={classes.appBar}>
-              <Tabs value={tab} onChange={handleTabChange} aria-label="Pattern Service" >
-                <Tab label="Settings" {...a11yProps(0)} />
+              <Tabs className={classes.appTabs} value={tab} onChange={handleTabChange} TabIndicatorProps={{
+                style : {
+                  display : "none",
+                },
+              }}  aria-label="Pattern Service" >
+                <Tab label={<div> <SettingsIcon style={{ verticalAlign : 'middle', transform : "scale(0.8)" }} />Settings</div>} {...a11yProps(0)} />
```

Comment:
Hey, @Chadha93 the changes are fine only one suggestion from my side is that maybe using the className would be better than inline styling.

Reply:
Oh, I missed that Thanks @sayantan1413 for this.